bot.py

```diff
@@ -162,6 +168,123 @@ async def get_last_match(ctx, name: str):
         print(e_values)
 
 
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+@bot.command(name="add", help="Add the players to the list")
+async def add_summoner(ctx, *, message):
+    """Writes list of summoners to local
+    json file and sends the list to the bot"""
+
+    try:
+        # typing indicator
+        async with ctx.typing():
+            await asyncio.sleep(1)
+
+        data_folder_path = get_file_path("data/")
+        json_path = data_folder_path + "data.json"
+
+        # create a directory containing json file to store data for added summoners
+        if not os.path.exists(json_path):
+            os.makedirs(data_folder_path)
+            with open(json_path, "w"):
+                pass
+
+        # converting the message into list of summoners
+        player_list = message.split(",")
+
+        # for importing data from json file
+        file_data = ""
+
+        # initializing total number of players for counting both incoming and existing summoners
+        total_number_of_players = 0
+
+        # initializing server id to a variable
+        server_id = str(ctx.guild.id)
+
+        # storing the json file into a variable
+        if os.path.getsize(json_path) > 0:
+            with open(json_path, "r") as file:
+                file_data = json.load(file)
+
+            # if server id exist in json, add number of players
+            if server_id in file_data:
+                total_number_of_players += len(file_data[server_id])
+
+            # remove summoners from incoming data if it exists in json file with same server ID
+            for player_name in player_list:
+                if any(
+                    player_name in player["user_name"]
+                    for player in file_data[server_id]
+                ):
+                    player_list.remove(player_name)
+
+        # add number of summoners from incoming data to total number of players
+        total_number_of_players += len(player_list)
+
+        if total_number_of_players > MAX_NUM_PLAYERS_TEAM:
+            raise Exception(
+                "Limit Exceeded", "You have exceeded a limit of 10 summoners!"
```

Comment:
I might be being little picky, but I liked previous way of returning how many more players needs to be added.
`num_of_players_needed = 10 - len(file_data[server_id])`